docs/model_structure.rst

```diff
@@ -10,43 +10,49 @@ Model structure
 
-Filenames for the components are of the form
-carrier-tech.csv (e.g. :file:`electricity-demand.csv`, :file:`gas-bpchp.csv`).
+.. csv-table::
+   :delim: ;
+   :file: ../oemof_b3/schema/scalars.csv
 
-There is a set of general variables that appear in all components:
+Time series
 
-* **region** Region of a component. (the regions are defined in the topology file
-  :file: ../oemof_b3/model/topology.yml)
-* **name** Unique name of the form :py:attr:`'region-carrier-tech'` (e.g. :py:attr:`'LU-gas-bpchp'`,
-  :py:attr:`'AT-electricity-airsourcehp'`).
-* **carrier** Energy carrier of the flow into the component (e.g. solar, wind, biomass, coal,
-  lignite, uranium, oil, gas, methane, hydro, waste, electricity, heat). This allows to categorize
-  the component and should correspond to the bus to which the component is connected.
-* **tech** More detailled specification of the technology (e.g. st, ocgt, ccgt, pv, onshore,
-  offshore, ror, phs,
-  extchp, bpchp, battery)
-* **type** Type of oemof.tabular.facade, defined in `TYPEMAP`.
+.. csv-table::
+   :delim: ;
+   :file: ../oemof_b3/schema/timeseries.csv
+
+A few more conventions are important to know. Missing data is left empty. If a value applies to all
+regions, this is indicated by :attr:`ALL`. If it applies to the sum of regions, by :attr:`TOTAL`.
+
+Preprocessed datapackages
+-------------------------
+
+The input data as described above is then preprocessed. The preprocessed data in turn is organized in
+elements (scalars data) and sequences, stored in separate folders and with one file for each component.
+Below is an example of the element file for the gas turbine of the base examples scenario, which can be found in
+:file:`examples/base/preprocessed/base/data/elements/ch4-gt.csv`.
+
+=======  =========  ==========  =======  =====  ========  ==============  ========  =============  ===========  =============  =============  ==========  =================
+region   name       type        carrier  tech   from_bus  to_bus          capacity  capacity_cost  efficiency   carrier_cost   marginal_cost  expandable  output_paramters
+=======  =========  ==========  =======  =====  ========  ==============  ========  =============  ===========  =============  =============  ==========  =================
+BE       BE-ch4-gt  conversion  ch4      gt     BE-ch4    BE-electricity  1500000                  0.619        0.021          0.0045         False       {}
+BB       BB-ch4-gt  conversion  ch4      gt     BB-ch4    BB-electricity  600000                   0.619        0.021          0.0045         False       {}
+=======  =========  ==========  =======  =====  ========  ==============  ========  =============  ===========  =============  =============  ==========  =================
 
 Beyond that, there are specific variables which depend on the type of the component. Components and
 their properties are defined in
 `oemoflex <https://github.com/rl-institut/oemoflex/tree/dev/oemoflex/model>`_.
```

Comment:
```suggestion
`oemoflex <https://github.com/rl-institut/oemoflex/tree/dev/oemoflex/model>`_. Components and properties and also be added or updated in oemof-B3 using the files in :file:`oemof_b3/model/`.
```

Reply:
This is really a helpful suggestion. It would be nice to explain further how to use these files in a future release.